dts/bindings/modem/u-blox,lara-r6.yaml

```diff
@@ -11,3 +11,10 @@ properties:
   mdm-power-gpios:
     type: phandle-array
     required: true
+
+  target-speed:
+    type: int
+    default: 3000000
```

Comment:
Isn't baud rate dependent on host capabilities? Might a more conservative default of `1000000` be better?

Reply:
Absolutely, and this was the primary motivation for the change. The value of 3000000 was chosen to match the deprecated kconfig option, but I can drop it back to 921600 or something if that is preferable.